### ISSUE TITLE
dont't work on Android Stulio

### ISSUE THREAD

**Issue by sapfir63** [jakepurple13/Full-Multiplatform-Compose-Plugin#9]
Hello
on Android Studio the project is not visible in New Project
or it does not work in Android Studio?

**Comment by naser09** [comment]
Android Studio Flamingo | 2022.2.1 Patch 2
same here .

**Comment by kalelidev** [comment]
Android Studio Flamingo | 2022.2.1 Patch 2 can´t show on the wizard.

**Comment by algoflows** [comment]
Yep doesn't seem to work in Android studio, what's the situation is this going to be sorted anytime soon?

**Comment by jakepurple13** [comment]
I've been quite busy and haven't had time to work on this.
Sorry. When I get some free time I'll look into this.

**Comment by algoflows** [comment]
Also does the web project default to the new wasm?

**Comment by MessiasLima** [comment]
The project creation in Android Studio is a bit "custom". It would be nice if there is any option in the menu for creating the project
